src/asmcnc/comms/router_machine.py

```diff
@@ -1204,14 +1204,20 @@ def g28_z(self): return float(self.s.g28_z)
 
 # SPEED AND FEED GETTERS
     def feed_rate(self): return int(self.s.feed_rate)
+
     def spindle_speed(self): 
         if self.spindle_voltage == 110: 
             # if not self.spindle_digital or not self.fw_can_operate_digital_spindle(): # this is only relevant much later on
             converted_speed = self.convert_from_230_to_110(self.s.spindle_speed)
             return int(converted_speed)
         else: 
             return int(self.s.spindle_speed)
```

Comment:
are explicit casts required?

Reply:
yep, it's complicated - will explain another time!